README.md

```diff
@@ -83,11 +87,16 @@ python tuning/sft_trainer.py  \
 ```
 
 ### Multiple GPUs with FSDP
+
```

Comment:
maybe taking lines 87-93 from here is a good idea, since fabian added them https://github.com/foundation-model-stack/fms-hf-tuning/pull/91/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R87

Reply:
Sure @Ssukriti, I have cherry picked this from Fabian's PR automatically credited him in this commit - https://github.com/foundation-model-stack/fms-hf-tuning/commit/4aa0153ab0c55ac5b305b8d60e8f00f89cc354d4